sqlbucket/cli.py

```diff
@@ -1,10 +1,18 @@
 import click
 from sqlbucket.utils import logger, n_days_ago, cli_variables_parser, success
 import sys
+import yaml
 
 
-def load_cli(sqlbucket_object):
+def every_dbs(env):
+    connections = yaml.load(
+        open(f'configs/{env}/connection_urls.yaml', 'r'),
+        Loader=yaml.FullLoader
+    )
+    return list(connections.keys())
```

Comment:
I really like the feature, but I'm not sure this is the right implementation.
The extraction from the config is only based on how you guys are currently using the library at SP. The f'configs/{env}/connection_urls.yaml' may be irrelevant in other organisations.

Instead of getting the connection names from your config, try to get it from the SQLBucket object. See here https://github.com/socialpoint-labs/sqlbucket/blob/master/sqlbucket/core.py#L26

if you get all your connections from your SQLBucket object instead, then it becomes agnostic on how everyone configure their connections in the first place.

Reply:
Makes sense :-)